podpac/core/coordinates/dependent_coordinates.py

```diff
@@ -492,6 +495,39 @@ def transpose(self, *dims, **kwargs):
             properties["dims"] = dims
             return DependentCoordinates(coordinates, **properties)
 
+    def issubset(self, other):
```

Comment:
I believe this is NEVER called in practice, since `Coordinates.issubset` loops through `Coordinates.values()` which gets you to the `ArrayCoordinatesNd.issubset` function instead...

Reply:
I haven't looked closely at that yet, but that doesn't really make sense to me.

Reply:
Meaning that if that is true, then I think there is a bug in `Coordinates.issubset`.

Reply:
Yeah I'm wrong. It is called.